lib/registries/addon/components/page-link/template.hbs

```diff
@@ -0,0 +1,7 @@
+{{yield (hash
+     validationClass=this.pageClass
```

Comment:
IMHO, we shouldn't pass around CSS class names as state. Instead, we should map state to CSS class. A good way to define possible states is with an enum:
```ts
export enum PageState {
    Unvisited,
    Active,
    Valid,
    Invalid,
}
```
and then `pageClass` would become `pageState` and return `PageState` instead of `string`:
```ts
    @computed('pageManager.{isVisited,pageIsValid}', 'pageIsActive')
    get pageState(): PageState {
        if (this.pageIsActive) {
            return PageState.Active;
        }
        if (this.pageManager.isVisited) {
            return this.pageManager.pageIsValid ? PageState.Valid : PageState.Invalid
        }
        return PageState.Unvisited;
    }
```
Then we'd map pageState to CSS class in the draft route. One way to do this is to create a local helper to map state to class using [compute](https://github.com/DockYard/ember-composable-helpers#compute). First we'd create an action in the draft controller to do the mapping:
```ts
import { PageState } from 'registries/components/page-link/component';

@action
pageStateToClass(pageStage: PageState): string {
    switch(pageState) {
    case PageState.Unvisited:
        return 'Unvisited'
}
```
and then use it as a helper in the draft template:
```hbs
<leftNav.link local-class={{compute (action this.pageStateToClass) pageLink.pageState}}
```
I know this seems like extra work, but it think the decoupling of CSS class from state is worth it.